src/options.rs

```diff
@@ -262,7 +294,7 @@ impl Options {
             Some("color") => OutputStyleOption::Color,
             Some("none") => OutputStyleOption::Disabled,
             _ => {
-                if options.command_output_policy == CommandOutputPolicy::Discard
+                if options.command_output_policy != CommandOutputPolicy::Forward
```

Comment:
Inverting this whole condition (and exchanging if and else blocks below) would make it a bit easier to read IMO:
```rust
if options.command_output_policy == CommandOutputPolicy::Forward || !atty::is(Stream::Stdout) {
  OutputStyleOption::Basic
} else {
  OutputStyleOption::Full
}
```